src/components/ec-navigation/ec-navigation.vue

```diff
@@ -53,30 +53,27 @@
   </div>
 </template>
 
-<script>
-export default {
-  name: 'EcNavigation',
-  props: {
-    isCollapsed: {
-      type: Boolean,
-      required: true,
-      default: false,
-    },
-    isCollapsable: {
-      type: Boolean,
-      required: true,
-    },
-    branding: {
-      type: Object,
-      default: () => ({}),
-      required: true,
-    },
-    showBrandingLogo: {
-      type: Boolean,
-      default: true,
-    },
+<script setup>
+defineProps({
+  isCollapsed: {
+    type: Boolean,
+    required: true,
+    default: false,
```

Comment:
"default" or "required" field should be removed, there is no reason to keep both simultaniously.